### PR TITLE
Individual posts

### PR DESCRIPTION
**Pull request by yashpatel5400**

### CODE REVIEW

File: app/assets/stylesheets/posting.css.scss [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+
+.location {
+	display: inline-block;
+	width: 150px;
+	height: 100px;
+	text-align: center;
+	vertical-align: middle;
+}
+
+.date {
```
Comment (izhan) [inline app/assets/stylesheets/posting.css.scss]: one thing i'd change is to make these css classes more specific to make sure there are no conflicts elsewhere in the app.  

i.e. 
`.date` -> `.posting-date`
`.location` -> `.posting-location`
etc

**Comment by izhan** [comment]
great PR.  i'd love to merge it after the class naming is fixed

**Comment by ksivakumar** [comment]
Yup, will get it to u before tonight!

**Comment by izhan** [comment]
Awesome thanks man!!

**Pull request merged**